### PR TITLE
fix failure of scaling up es pods when running CI testing

### PR DESCRIPTION
**Pull request by QiaolingTang**
This scenario always failed when running CI testing, the ES pods are pending due to limited resources, so I update the script to only check the new ES deployment is created or not, don't check the pod status. @anpingli PTAL, thanks.

### CODE REVIEW

File: features/logging/cluster_logging_operator.feature [hunk 1]
```diff
@@ -56,33 +56,47 @@ Feature: cluster-logging-operator related test
...
+    # Given admin ensures "instance" cluster_logging is deleted from the "openshift-logging" project after scenario
+    And I wait for the "elasticsearch" elasticsearches to appear
     And the expression should be true> cluster_logging('instance').logstore_node_count == 2
-    Given 2 pods become ready with labels:
-      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
+    And the expression should be true> elasticsearch('elasticsearch').nodes[0]['nodeCount'] == 2
+    Given evaluation of `elasticsearch('elasticsearch').nodes[0]['genUUID']` is stored in the :gen_uuid_1 clipboard
+    Then I wait for the "elasticsearch-cdm-<%= cb.gen_uuid_1 %>-1" deployment to appear
+    And I wait for the "elasticsearch-cdm-<%= cb.gen_uuid_1 %>-2" deployment to appear
+    Then the step should succeed
```
Comment (pruan-rht) [inline features/logging/cluster_logging_operator.feature]: @QiaolingTang you can remove this step.

File: features/logging/cluster_logging_operator.feature [hunk 2]
```diff
@@ -56,33 +56,47 @@ Feature: cluster-logging-operator related test
...
     Then the step should succeed
     And the expression should be true> cluster_logging('instance').logstore_node_count == 3
-    And 3 pods become ready with labels:
-      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
+    Given I wait for the steps to pass:
+    """
+    And the expression should be true> elasticsearch('elasticsearch').nodes[0]['nodeCount'] == 3
+    """
+    And I wait for the "elasticsearch-cdm-<%= cb.gen_uuid_1%>-3" deployment to appear
+    Then the step should succeed
```
Comment (pruan-rht) [inline features/logging/cluster_logging_operator.feature]: @QiaolingTang you can remove this step.

File: features/logging/cluster_logging_operator.feature [hunk 3]
```diff
@@ -56,33 +56,47 @@ Feature: cluster-logging-operator related test
...
-      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=true |
-    And a pod becomes ready with labels:
-      | cluster-name=elasticsearch,component=elasticsearch,es-node-client=true,es-node-data=true,es-node-master=false |
+    Given I wait for the steps to pass:
+    """
+    And the expression should be true> elasticsearch('elasticsearch').nodes[0]['nodeCount'] + elasticsearch('elasticsearch').nodes[1]['nodeCount'] == 4
+    """
+    Given evaluation of `elasticsearch('elasticsearch').nodes[1]['genUUID']` is stored in the :gen_uuid_2 clipboard
+    And I wait for the "elasticsearch-cd-<%= cb.gen_uuid_2 %>-1" deployment to appear
+    Then the step should succeed
```
Comment (pruan-rht) [inline features/logging/cluster_logging_operator.feature]: @QiaolingTang you can remove this step.

**Comment by QiaolingTang** [comment]
@pruan-rht I've updated my feature file per your comments, could you please help to review this PR again? Thanks a lot.

**Pull request merged**